tests/gis_tests/geoapp/tests.py

```diff
@@ -900,3 +902,16 @@ def test_non_concrete_field(self):
     def test_values_srid(self):
         for c, v in zip(City.objects.all(), City.objects.values()):
             self.assertEqual(c.point.srid, v['point'].srid)
+
+    def test_geometry_value_annotation(self):
+        p = Point(1, 1, srid=4326)
+        point = City.objects.annotate(p=Value(p, GeometryField(srid=4326))).first().p
+        self.assertEqual(point, p)
+
+    @skipUnlessDBFeature("supports_transform")
+    def test_geometry_value_annotation_different_srid(self):
+        tol = 10 ** -5
+        p = Point(1, 1, srid=32140)
+        point = City.objects.annotate(p=Value(p, GeometryField(srid=4326))).first().p
+        self.assertTrue(point.equals_exact(p.transform(4326, clone=True), tol))
```

Comment:
Since `tol` is only used once, is inlining `tolerance=1e-5` okay (rather than a separate variable)?